Makefile

```diff
@@ -69,6 +73,15 @@ $(TESTS) $(TOOLS) $(EXAMPLES): $(CCTZ_LIB)
 
 $(TESTS:=.o) $(TOOLS:=.o) $(EXAMPLES:=.o):
 
+$(CCTZ_SHARED_LIB): $(CCTZ_OBJS)
+	$(CC) $(LDFLAGS) -shared -Wl,-soname,$(CCTZ_SHARED_SONAME) -o$@ $(CCTZ_OBJS)
+
+$(CCTZ_SHARED_SONAME): $(CCTZ_SHARED_LIB)
```

Comment:
Hmm, having these rules makes it a bit easier to build the library as you can do `make libcctz.so` rather than having to know the version number (`make libcctz.so.2.0`). I've left them in but I can remove them if you don't see that as a benefit.